tests/patterns/python/cp_label.py

```diff
@@ -11,7 +11,7 @@
     #ERROR:
     "foo"
 
-#OK:
+#ERROR: a is foo by constant propagation
```

Comment:
Given the nature of the test I would have changed this to:
```python
#OK:
a = "bar"
```

Reply:
Done.